docker-compose.yml

```diff
@@ -0,0 +1,22 @@
+version: "3"
+services:
+
+    redaxo:
+        image: friendsofredaxo/redaxo:5
+        ports:
+            - ${REDAXO_PORT:-80}:80
+        volumes:
+            - .:/var/www/html:cached
+
+    db:
+        image: mysql:8
+        volumes:
+            - db:/var/lib/mysql:cached
+        environment:
+            MYSQL_DATABASE: redaxo
+            MYSQL_USER: redaxo
+            MYSQL_PASSWORD: redaxo   # do not use for production!
+            MYSQL_RANDOM_ROOT_PASSWORD: 'yes'
+
+volumes:
+    db:
```

Comment:
würde ggf. hier ein mapping in einen ordner machen wollen, da ich i.d.R. alle volumes mit `docker-compose down -v` direkt mit wegnehme.

Reply:
Könnten wir so machen, nur war ich mir unsicher, ob man wirklich standardmäßig einen neuen DB-Ordner im Repo haben will. Letztendlich fand ich es besser, das nicht zu machen, sondern das Volume zu verwenden. So machen es viele Docker-Rezepte standardmäßig, damit man sich nicht ums lokale Dateisystem kümmern muss.

`docker-compose down` solltest du eigentlich in der Praxis eher selten nutzen. Normalerweise stoppt man Container ja nur (`docker-compose stop`), um später schnell starten und nahtlos weiter dran arbeiten zu können.

Jedenfalls, ich würde tendenziell eher bei der aktuellen Volume-Variante bleiben wollen, richte mich aber gerne auch nach euren Meinungen.

Reply:
ok, dann lassen wir es so.